aiohttp/resolver.py

```diff
@@ -0,0 +1,72 @@
+import socket
+import asyncio
+from .abc import AbstractResolver
+
+try:
+    import aiodns
+except ImportError:
+    aiodns = None
+
+
+class ExecutorResolver(AbstractResolver):
```

Comment:
Strictly speaking thread pool is just internal details of asyncio implementation, and may be change in future. I'd call it 'DefaultResolver' and document it as default asyncio based resolver